manifests/params.pp

```diff
@@ -40,8 +40,8 @@
                           }]
 
   $host            =     '0.0.0.0'
-  $port            =     3000
+  $port            =     '3000'
   $user            =     ''
   $pass            =     ''
-  $refresh         =     5
+  $refresh         =     '5'
```

Comment:
On one hand, converting to strings allows `validate_string` to work, but it might be unintuitive to a user. I would personally have not quoted these values if I had to specify them.

Instead of turning these values into strings, they can be left as parameters and instead of using `validate_re`, something like the following can be done:

```puppet
  if is_integer($port) == false {
    fail("The port parameter must be an integer. The current value is ${port}.")
  }

  if is_integer($refresh) == false {
    fail("The refresh parameter must be an integer. The current value is ${refresh}.")
  }
```

Or to make everyone happy, something like:

```puppet
  if is_integer($port) == false or $port =~ /\d+/ {
    fail("The port parameter must be an integer. The current value is ${port}.")
  }

  if is_integer($refresh) == false or $refresh =~ /\d+/ {
    fail("The refresh parameter must be an integer. The current value is ${refresh}.")
  }
```

(I haven't verified that last example works)